share_extension/android/extension_channels/extension_channels.js

```diff
@@ -45,10 +52,12 @@ export default class ExtensionTeam extends PureComponent {
 
     state = {
         sections: null,
+        loading: true,
     };
 
     componentDidMount() {
         this.buildSections();
+        this.searchChannels('');
```

Comment:
I'm not 100% sure, but won't `buildSections` get called from within the new call to `searchChannels('')` ?

Wondering why this initialization call to `buildSections` is still necessary.

Reply:
The first `buildSections` call is to lay out the channels that are immediately available in the redux store. `searchChannels` is then called to find the remaining channels and then `buildSections` gets called again thereafter with the new data. The jira ticket cited this type of flow to account for potential loading times when fetching these extra channels.